manager/src/main/java/com/redhat/service/bridge/manager/api/user/ProcessorsAPI.java

```diff
@@ -47,39 +50,49 @@
     @Inject
     ProcessorService processorService;
 
+    @Inject
+    BridgesService bridgesService;
+
     @Inject
     IdentityResolver identityResolver;
 
     @Inject
     JsonWebToken jwt;
 
     @GET
-    @Path("{bridgeId}/processors/{processorId}")
-    public Response getProcessor(@NotEmpty @PathParam("bridgeId") String bridgeId, @NotEmpty @PathParam("processorId") String processorId) {
+    @Path("{bridgeIdentifier}/processors/{processorId}")
+    public Response getProcessor(@NotEmpty @PathParam("bridgeIdentifier") String bridgeIdentifier, @NotEmpty @PathParam("processorId") String processorId) {
```

Comment:
`bridgeIdentifier` is a bit misleading, `idOrName` would be better I guess

Reply:
+1 to `idOrName` as it is self-describing.

Reply:
done